tools/clang/test/CodeGenSPIRV/passthru-vs.hlsl2spv

```diff
@@ -0,0 +1,49 @@
+// Run: %dxc -T vs_6_0 -E VSmain
+
+/*
+// This is the proper way to do it, but our code doesn't handle PSInput
+// as return type for now.
+
+struct PSInput {
+  float4 position : POSITION;
+  float4 color : COLOR;
+};
+
+PSInput VSmain(float4 position: POSITION, float4 color: COLOR) {
+  PSInput result;
+  result.position = position;
+  result.color = color;
+  return result;
+}
+*/
+
+float4 VSmain(float4 position: POSITION, float4 color: COLOR) {
+  return color;
+}
+
+// TODO:
+// Proper representation for the structure.
+// Input/Output interfaces for OpEntryPoint
+// Proper logic to determine ExecutionMode
```

Comment:
Yep, it's reasonable to put TODO for execution mode here since it's wrong in the following. We should not generate `OpExecutionMode` for this shader.